src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -46,5 +46,11 @@ internal class ConvertOptions
             "plugin-assembly-path",
             HelpText = "Path to plugin assembly containing converter types.")]
         public string PluginAssemblyPath { get; internal set; }
+
+
+        [Option(
```

Comment:
>( [](start = 15, length = 1)

Hadn't you already added this option in a previous PR, and didn't you use `'f'` as the short name, per my comment? Between this an my comment on SetCurrentVersion.cmd, I wonder if the branch you're working in is synced to latest changes.